plugins/inputs/smart/smart.go

```diff
@@ -491,12 +935,30 @@ func parseTemperature(fields, deviceFields map[string]interface{}, str string) e
 	return nil
 }
 
+func parseTemperatureSensor(fields, deviceFields map[string]interface{}, str string) error {
+	var temp int64
+	if _, err := fmt.Sscanf(str, "%d C", &temp); err != nil {
+		return err
+	}
+
+	fields["raw_value"] = temp
+
+	return nil
+}
+
 func init() {
+	// Set LC_NUMERIC to uniform numeric output from cli tools
+	_ = os.Setenv("LC_NUMERIC", "en_US.UTF-8")
```

Comment:
a bit odd, but I imagine it's fine.

Reply:
Format of `smartctl` and `nvme` outputs could be based on `LC_NUMERIC` value. Setting it to `en_US.UTF-8` ensures us that it can be easily parsable independently from local settings in system.

Reply:
Format of `smartctl` and `nvme` outputs could be based on `LC_NUMERIC` value. Setting it to `en_US.UTF-8` ensures us that it can be easily parsable independently from local settings in system.